### ISSUE TITLE
TextBoxDirective - Material theme broken

### ISSUE THREAD

**Issue by cluetjen** [telerik/kendo-angular#3567]
`<input kendoTextBox />` doesn't look like a material component and looks different to `&lt;kendo-textbox />´.

![image](https://user-images.githubusercontent.com/5447876/152045769-18ee0913-26f8-429a-af01-2fa6b0efb6c7.png)
https://www.telerik.com/kendo-angular-ui/components/inputs/textbox/textbox-directive/

Expected:

![image](https://user-images.githubusercontent.com/5447876/152045846-d32b622a-af6b-49c5-8745-407ba39b0b22.png)
https://www.telerik.com/kendo-angular-ui/components/inputs/textbox/

Please ignore the different configuration of the examples and focus on borders:
* Wrong: border top, right, bottom, left
* Expected: border bottom only

**Comment by mbechev** [comment]
Seems related to https://github.com/telerik/kendo-angular/issues/3566

**Comment by cluetjen** [comment]
@mbechev - for the records: Related: yes. Duplicate: no. #3566 = text box theme changed. This one: Even the changed theme is not applied correctly for textBoxDirective.

**Comment by mbechev** [comment]
The same layout is reproduced with the [kendoTextArea](https://www.telerik.com/kendo-angular-ui/components/inputs/textarea/textarea-directive/) directive:

https://stackblitz.com/edit/angular-dslkfg

**Comment by Raisolution** [comment]
Fixes for both the TextBox and TextArea directives are available in `v8.0.5`

**Comment by mpoqq** [comment]
@Raisolution the TextArea directive is still not looking like the TextArea component. The directive has no padding.

```
<textarea [style.width.px]="200" kendoTextArea placeholder="Enter your name"></textarea>
<br>
<kendo-textarea [style.width.px]="200" placeholder="Enter your name"></kendo-textarea>
```
![grafik](https://user-images.githubusercontent.com/20458956/154698692-01333dad-5b5b-415a-8348-d2018f9426ec.png)
